pytest.ini

```diff
@@ -1,2 +1,134 @@
 [pytest]
 norecursedirs = .git build ci dist extern release tools unit venv
+pep8ignore =
+    * E111 E114 E115 E116 E121 E122 E123 E124 E125 E126 E127 E128 E129 E131 E226 E240 E241 E242 E243 E244 E245 E246 E247 E248 E249 E265 E266 E704 W503
```

Comment:
This line ignores errors based on our [existing config](https://github.com/matplotlib/matplotlib/pull/8001/files#diff-13006e1a4da2e47c3f768fdd0f2cfaebL19) + `pep8`'s default ignore list. I tried not ignoring these globally, but they apply to a _lot_ of files, so it's very difficult to limit this to just the files that need it.

The files we don't expect to pass are already listed below with the specific ignores.